src/main/kotlin/HttpStatus.kt

```diff
@@ -1,13 +1,14 @@
 package httpstatus
 
 enum class HttpStatus(val value: Int) {
-    OK(200), BadRequest(400), NotFound(404), MovedPermanently(301), InternalServerError(500);
+    OK(200), BadRequest(400), NotFound(404), MovedPermanently(301),
+    NotAllowed(405);
```

Comment:
It'd be a nice touch if these were in numeric order.

Reply:
Organized status codes in numeric order